client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -126,18 +126,22 @@ class ImageEditorCanvas extends Component {
 
 		this.drawImage();
 		this.updateCanvasPosition();
-		this.onWindowResize = throttle( this.updateCanvasPosition, 200 );
+
 		if ( typeof window !== 'undefined' ) {
 			window.addEventListener( 'resize', this.onWindowResize );
 		}
 
 		this.props.setImageEditorImageHasLoaded( this.image.width, this.image.height );
 	}
 
+	onWindowResize() {
+		this.requestAnimationFrameId = window.requestAnimationFrame( this.updateCanvasPosition );
```

Comment:
I'm thinking we can still debounce this or add an early return in updateCanvasPosition if we've called it again too soon.